labs/shipped-application-level-policy/4.md

```diff
@@ -0,0 +1,26 @@
+## aie-burnit Usage
+
+We have deployed app up and running on test env.
+
+![](assets/13.PNG)
+
+We can specify the no of MB's by which we can start memory leak in app to see how policy response to our app.
+
+So we are adding 2 and click on update button, once done we will click on resume (it resume app with memory leak.)
+
+![](assets/14.PNG)
+
+As we can see memory leak has been increasing. As Memory leak reached 75% of policy, we will get alert, in shipped ui.
+
+We can see alert under Run page > 
+
+![](assets/15.PNG)
+
+After reach to threshold we have stopped memory leak and capture the below output.
```

Comment:
Here we we need to highlight 
2/2 instances are running and service scaled up from 1-2 instances.